### ISSUE TITLE
Request for creating a Security Advisory

### ISSUE THREAD

**Issue by stypr** [inunosinsi/soycms#17]
## Summary

JPCERT/CC asked me to directly contact here to create a CVE number.
Can you do the following steps to assign a CVE number?

For issues: https://github.com/inunosinsi/soycms/issues/7, https://github.com/inunosinsi/soycms/issues/8, https://github.com/inunosinsi/soycms/issues/9, https://github.com/inunosinsi/soycms/issues/10

## Steps

1. Access `Security` Tab of the page
![https://docs.github.com/assets/images/help/repository/security-tab.png](https://docs.github.com/assets/images/help/repository/security-tab.png)

2. Click `Security Advisory` Tab
![https://docs.github.com/assets/images/help/security/advisories-tab.png](https://docs.github.com/assets/images/help/security/advisories-tab.png)

3. Click `New Draft Security Advisory`
![https://docs.github.com/assets/images/help/security/security-advisory-new-draft-security-advisory-button.png](https://docs.github.com/assets/images/help/security/security-advisory-new-draft-security-advisory-button.png)

4. Write Contents (You can just copy-paste from issues.)
![https://github.com/stypr/advisory-share/blob/master/writing_advisory.png?raw=true](https://github.com/stypr/advisory-share/blob/master/writing_advisory.png?raw=true)

5. Add collaborator: `stypr`
![https://github.com/stypr/advisory-share/blob/master/writing_advisory_3.png?raw=true](https://github.com/stypr/advisory-share/blob/master/writing_advisory_3.png?raw=true)

5. Finally, Click `Request CVE`
![https://github.com/stypr/advisory-share/blob/master/writing_advisory_2.png?raw=true](https://github.com/stypr/advisory-share/blob/master/writing_advisory_2.png?raw=true)

**Comment by stypr** [comment]
@inunosinsi 

Thanks for the check.
But, I see that there is only 1 request.
could you please post other 3 as well?

**Comment by stypr** [comment]
Title: Reflected XSS leading to RCE in SoyCMS
Severity: High
Affected Versions: < 3.0.2.328
Patched Versions: 3.0.2.328

```
SoyCMS 3.0.2 and earlier is affected by Reflected Cross-Site Scripting (XSS) which leads to Remote Code Execution (RCE) from a known vulnerability. This allows remote attackers to force the administrator to edit files once the adminsitrator loads a specially crafted webpage.

Impact: XSS to RCE via Inquiry Error and Content modification

* Attack vector is: Administrator must be logged in.
* Components are: CMSApplication.class.php
* Tested SoyCMS Version :  3.0.2 (latest)
* Affected SoyCMS Version : ~3.0.2

Found by @stypr from Vulnerability Research Team in [Flatt Security Inc.](https://flatt.tech/)

Full Exploit Video: https://youtu.be/uAMAwH35ups
```
---

Title: Remote Code Execution in SoyCMS
Severity: Medium
Affected Versions: < 3.0.2.328
Patched Versions: 3.0.2.328

```
SoyCMS 3.0.2 and earlier is affected by Remote Code Execution (RCE) using Unrestricted File Upload. Cross-Site Scripting(XSS) vulnerability that was reported earlier can be chained in order to perform a successful remote code execution by redirecting the administrator to load a specially crafted webpage.

Impact: XSS to RCE via Inquiry Error and Unrestricted File Upload

* Attack vector is: Administrator must be logged in.
* Components are: File Manager
* Tested SoyCMS Version :  3.0.2 (latest)
* Affected SoyCMS Version : ~3.0.2

Found by @stypr from Vulnerability Research Team in [Flatt Security Inc.](https://flatt.tech/)

Full Exploit Video: https://youtu.be/FWIDFNXmr9g
```

---

Title: Cross-site Request Forgery (CSRF) leading to RCE in SoyCMS
Severity: Medium
Affected Versions: < 3.0.2.328
Patched Versions: 3.0.2.328

```
SoyCMS 3.0.2 and earlier is affected by Cross-site Request Forgery (CSRF) and Remote Code Execution (RCE). This allows remote attackers to force the administrator to edit files once the adminsitrator loads a specially crafted webpage.

Impact: CSRF to RCE via Unrestricted File Modification

* Attack vector is: Administrator must be logged in.
* Components are: Soy Inquiry
* Tested SoyCMS Version :  3.0.2 (latest)
* Affected SoyCMS Version : ~3.0.2

Found by @stypr from Vulnerability Research Team in [Flatt Security Inc.](https://flatt.tech/)

Full Exploit Video: https://youtu.be/ffvKH3gwyRE
```

**Comment by inunosinsi** [comment]
@stypr 
thank you for creating the text.
